examples/redshift/connector.py

```diff
@@ -0,0 +1,176 @@
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+import json  # Import the json module to handle JSON data.
+# Import datetime for handling date and time conversions.
+from datetime import datetime
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+import redshift_connector
+from fivetran_connector_sdk import Logging as log
+from fivetran_connector_sdk import Operations as op
+
+TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+# This column is the replication key which helps determine which records are updated
+REPLICATION_KEY = "updated_at"
+
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function:
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+def schema(configuration: dict):
+    return [
+        {
+            "table": "customers",  # Name of the table in the destination.
+            "primary_key": ["customer_id"],  # Primary key column(s) for the table.
+            "columns": {  # Define the columns and their data types.
+                "customer_id": "INT",  # Integer column for the customer_id.
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "email": "STRING",  # String column for the email.
+                "updated_at": "UTC_DATETIME",  # UTC date-time column for the updated_at.
+                # In this example we are using `updated_at` as the replication key
+            },
+        }
+    ]
+
+
+def dt2str(incoming: datetime) -> str:
+    return incoming.strftime(TIMESTAMP_FORMAT)
+
+
+def connect_to_redshift(configuration):
+    return redshift_connector.connect(
+        host=configuration['host'],
+        database=configuration['database'],
+        port=configuration['port'],
+        user=configuration['user'],
+        password=configuration['password']
+    )
+
+
+def setup_db(cursor):
+    cursor.execute("CREATE SCHEMA IF NOT EXISTS testers;")
+    cursor.execute("DROP TABLE IF EXISTS testers.customers;")
+
+    cursor.execute("CREATE TABLE IF NOT EXISTS testers.customers "
+                   "(customer_id INTEGER NOT NULL, "
+                   "first_name VARCHAR, "
+                   "last_name VARCHAR, "
+                   "email VARCHAR, "
+                   "city VARCHAR, "
+                   "state VARCHAR, "
+                   "country VARCHAR, "
+                   "updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP)"
+                   " DISTSTYLE KEY DISTKEY (customer_id) SORTKEY (updated_at)")
```

Comment:
I think we need to add a primary key constraint on `customer_id`. 
Also, a small description of `DISTSTYLE KEY` to describe usage.

Reply:
Added PK
Removed DISTSTYLE KEY